packages/integration/src/gerrit/core.ts

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import { logger } from '@azure/identity';
```

Comment:
This is not our logger 😁, if logging was needed we'd want to inject the `LoggerService` from `@backstage/backend-plugin-api` instead